packages/client/subscriptions/TeamSubscription.ts

```diff
@@ -80,6 +80,13 @@ const subscription = graphql`
       ...UpdateTeamNameMutation_team @relay(mask: false)
       ...UpdateUserProfileMutation_team @relay(mask: false)
       ...UpgradeToProMutation_team @relay(mask: false)
+
+      ... on JiraIssue {
+        id
```

Comment:
thinking about it more-- yep, since 1 user is requesting this, the update should be pushed to that same user only

Reply:
@mattkrick doesn't this get called when we are in the estimate phase of a poker meeting too, which means all team members in the meeting should be able to see the updated integrated issue they are scoping?

Reply:
@mattkrick i just tested pushing the update on `NotificationSubscription` instead of `TeamSubscription` channel. when doing it this way, the integrated user will see the updated issue when pushed async but the other teammate won't. this is bc we would only push to the userId who owns the atlassian auth. this behavior seems un-inuitive to me, so you can confirm if this is correct or not?

<img width="932" alt="Screen Shot 2021-12-03 at 4 19 31 PM" src="https://user-images.githubusercontent.com/11562889/144689006-46dafd3c-256e-43a1-b2d9-b2988da7295a.png">

Reply:
ok i just pushed a commit changing it to publish on NotificationSubscription. If there is another teammate, they won't get the update if you refresh and get a fresher issue. but they'll get the fresher issue from the cache if they refresh their page. i think this is probably okay behavior.